chyt/README.md

```diff
@@ -0,0 +1,14 @@
+#### CHYT powered by ClickHouse
```

Comment:
"CHYT" ... what does that mean? Should it simply be "YTsaurus"?

Reply:
YTsaurus is a data platform. CHYT is high-level computing engine inside YTsaurus [architecture](https://ytsaurus.tech/docs/en/overview/about).